dotCMS/src/main/java/com/dotcms/storage/ObjectReaderDelegate.java

```diff
@@ -0,0 +1,19 @@
+package com.dotcms.storage;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * Simple method to read an object into the stream
+ * @author jsanca
+ */
+@FunctionalInterface
+public interface ObjectReaderDelegate {
+
+    /**
+     * Writes the object into the stream
+     * @param stream {@link InputStream}
+     * @return  object {@link Object}
+     */
+    public Object read(final InputStream stream);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifier 'public' on method 'read': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5547873)